### PR TITLE
Engine support update

### PR DESCRIPTION
**Pull request by josephwright**
**READ ME FIRST**: Please understand that in most cases we will not be able to merge a pull request because there are a lot of internal activities needed when updating the LaTeX2e sources. If you have a code suggestion please discuss it with the team first.

# Internal housekeeping

## Status of pull request

- Feedback wanted 
- Under development
- [x] Ready to merge

## Checklist of required changes before merge will be approved
- [ ] Test file(s) added
- [ ] Version and date string updated in changed source files
- [ ] Relevant `\changes` entries in source included
- [ ] Relevant `changes.txt` updated
- [ ] Rollback provided (if necessary)?
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/doc/ltnews40.tex [hunk 1]
```diff
@@ -159,6 +159,26 @@ \section{Introduction}
...
 
+\section{Engine support: important update}
+
+As detailed below, work is progressing on the Tagged PDF project. There are
+many drivers for this work, but include legal changes in many places which will
+increasing require well-tagged PDFs including full support for mathematics. As
+part of the work on this, we are looking at the technical abilities of the
+\TeX{} engines.
+
+With for \XeTeX{}, it is impossible to reliably produce tagged PDFs due to
```
Comment (zauguin) [inline base/doc/ltnews40.tex:170]:
```suggestion
With \XeTeX{}, it is impossible to reliably produce tagged PDFs due to
```

File: base/doc/ltnews40.tex [hunk 2]
```diff
@@ -159,6 +159,26 @@ \section{Introduction}
 
 \emph{to write \ldots\ 30th aniversary of \LaTeXe{} this year btw}
 
+\section{Engine support: important update}
+
+As detailed below, work is progressing on the Tagged PDF project. There are
+many drivers for this work, but include legal changes in many places which will
+increasing require well-tagged PDFs including full support for mathematics. As
```
Comment (Skillmon) [inline base/doc/ltnews40.tex:166]: Something's off in that sentence after the comma. Maybe
```suggestion
many drivers for this work, including legal changes in many places which will
increasingly require well-tagged PDFs including full support for mathematics. As
```

**Review by Skillmon — COMMENTED** [review]
This covers the 3 major engines in the western world. Anything regarding (u)pTeX planned to be added here?

**Comment by josephwright** [comment]
> This covers the 3 major engines in the western world. Anything regarding (u)pTeX planned to be added here?

We've never formally supported (u)pTeX - they are not in the test suite and (u)pLaTeX contain a significant amount of code maintained and tested by an independent group. So in that sense there is nothing to say. The pressure to tag all-Japanese documents are not known to us, so it's very hard to say more.

File: base/doc/ltnews40.tex [hunk 3]
```diff
@@ -159,6 +159,28 @@ \section{Introduction}
...
+No \emph{new} functionality will be added for \XeTeX{} by the \LaTeX{} team. It
+is likely that over time functionality may become more restricted, and users
+are urged to migrate \XeTeX{} documents to \LuaTeX{}.
+
+For \pdfTeX{}, tagging is available and we are able to support mathematics by
+including \TeX{} mathematical sources or by using externally-generated MathML.
+Only \LuaTeX{} is capable of \emph{automatic} generation of MathML as part of a
+\LaTeX{} run. Thus \pdfTeX{} continues to be supported for existing material,
+but for new documents, moving to \LuaTeX{} is recommended.
+
```
Comment (FrankMittelbach) [inline base/doc/ltnews40.tex:183]: One could perhaps add a statement like

We cannot make statements about the support for other engines such as ... , as we don't use these programs nor have indepth knowledge of their functionalities. To the best of our knowledge, core LaTeX works well with these engines, but if and to what extent tagging can be supported will remain to be seen. If relevant information becomes available to us we will provide an update in future editions of the \LaTeX{} newsletter.

**Review by FrankMittelbach — APPROVED** [review]
with (perhaps) a suggested addition

**Pull request merged**